Library/Homebrew/cmd/completions.rb

```diff
@@ -0,0 +1,52 @@
+# typed: true
+# frozen_string_literal: true
+
+require "cli/parser"
+require "completions"
+
+module Homebrew
+  extend T::Sig
+
+  module_function
+
+  sig { returns(CLI::Parser) }
+  def completions_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `completions` [<subcommand>]
+
+        Control whether Homebrew automatically links shell  files.
```

Comment:
```suggestion
        Control whether Homebrew automatically links shell files.
```

Reply:
Oops, this is actually supposed to be:

```suggestion
        Control whether Homebrew automatically links shell completion files.
```